### ISSUE TITLE
fit page still has a few redraw issues (Trac #488)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#622]
The compute and fit buttons can overlap sometimes when various panels change size in response to user action (like add dQ etc).  Also the bottom of the scroll panel sometimes does not get resized when the panel's total size shrinks in response to user actions.

Migrated from http://trac.sasview.org/ticket/488
```json
{
    "status": "closed",
    "changetime": "2015-12-08T16:23:30",
    "_ts": "2015-12-08 16:23:30.221508+00:00",
    "description": "The compute and fit buttons can overlap sometimes when various panels change size in response to user action (like add dQ etc).  Also the bottom of the scroll panel sometimes does not get resized when the panel's total size shrinks in response to user actions.",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2015-11-29T00:27:06",
    "component": "SasView",
    "summary": "fit page still has a few redraw issues",
    "priority": "minor",
    "keywords": "",
    "milestone": "SasView 3.1.2",
    "owner": "butler",
    "type": "defect"
}
```

**Comment by butlerpd** [comment]
Trac update at `2015/11/29 01:44:53`: **butler** commented:

Fixed by replacing a couple of self.Layout() with self.!FitInside. For complex panels with many sizers changing size on user input this seems to be required to make the whole panel redraw.  This is in commit 1c2bf901c33eeec26176d8cf085ab77daf74075b which did not automatically close the ticket because I used fixes ticket #622 instead of just fixes #622

Should check Mac build before closing anyway

**Comment by ajj** [comment]
Trac update at `2015/12/08 16:23:30`: 
* **ajj** changed resolution from "" to "fixed"
* **ajj** changed status from "new" to "closed"
